pkg/interceptors/interceptors.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package interceptors
+
+import (
+	"net/http"
+
+	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+)
+
+// InterceptorInterface is the interface that all interceptors implement.
+type InterceptorInterface interface {
```

Comment:
Could this just be Interface instead of InterceptorInterface?

Reply:
Or how about just `Interceptor`?

Reply:
Done!